test/integration-test/Diagnostics/err_uninitialized_temporary.cpp

```diff
@@ -1,36 +0,0 @@
-//===--------------------------------------------------------------------------------*- C++ -*-===//
-//                         _       _
-//                        | |     | |
-//                    __ _| |_ ___| | __ _ _ __   __ _
-//                   / _` | __/ __| |/ _` | '_ \ / _` |
-//                  | (_| | || (__| | (_| | | | | (_| |
-//                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
-//                    __/ |                       __/ |
-//                   |___/                       |___/
-//
-//
-//  This file is distributed under the MIT License (MIT).
-//  See LICENSE.txt for details.
-//
-//===------------------------------------------------------------------------------------------===//
-
-// RUN: %gtclang% %file% -fno-codegen
```

Comment:
rather than removing, that will loose this code that we would still like to recover in the future, why dont you move it to an "disabled" folder, so that the tests do not trigger it?

Reply:
I was thinking about doing that. Here is why I've decided against it:

- This is the most basic test (1 read 1 write) and can be created in <2 min. Storing this somewhere in the file-structure seems unnecessary
- Once we have the pass properly set up we should do more thorough tests: Test intervals, test inter-stencil temporaries, test conditionals etc. These tests should have much more coverage and will likely need to be designed from scratch anyways
- In oder to "save" what we have as a reference, the issue to create new tests specifically links to this commit so we can see what we had before.

Thoughts?

Reply:
ok